src/Core/Mesh/TopologicalTriMesh/TopologicalMesh.hpp

```diff
@@ -49,6 +45,9 @@ class RA_CORE_API TopologicalMesh : public OpenMesh::PolyMesh_ArrayKernelT<Topol
     std::vector<OpenMesh::HPropHandleT<Vector3>> m_vec3Pph;
     std::vector<OpenMesh::HPropHandleT<Vector4>> m_vec4Pph;
 
+    friend class LoopSubdivider;
+    friend class CatmullClarkSubdivider;
```

Comment:
Could you please explain why using friend classes here ?
Does it mean you don't have access to the fields you want ?
This would prevents users to add other schemes without modifying Radium's source code.

Reply:
Yes, I cannot access the properties otherwise.
Should we give access to these? I'm not sure, @dlyr what do you think?

Reply:
You mean `m_vec3Pph` for instance ?
It looks reasonable to give access to all the attributes.